src/api/db/data/20200420080753_generate_web_notifications.rb

```diff
@@ -0,0 +1,44 @@
+class GenerateWebNotifications < ActiveRecord::Migration[6.0]
+  def up
+    generate_web_notifications
+    generate_event_subscripitions
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
+  end
+
+  private
+
+  def generate_web_notifications
+    Notification::RssFeedItem.all.each do |rss_notification|
+      duplicate = rss_notification.dup
+      duplicate.type = 'Notification::WebItem'
+      # to avoid duplicated web notifications
+      Notification::WebItem.find_or_create_by(duplicate.attributes.except('id'))
+    end
+  end
+
+  def generate_event_subscripitions
+    event_subscriptions = EventSubscription.where(channel: :instant_email).where.not(user_id: nil)
+    event_subscriptions.each do |event_subscription|
+      create_subscription_for_channel(event_subscription, :rss)
+      create_subscription_for_channel(event_subscription, :web)
+    end
+  end
```

Comment:
What about the global defaults? `EventSubscription.where(channel: :instant_email).where(user_id: nil)`

Reply:
cannot we enforce the relationship between Notification and an user and then get rid of this last where?